### ISSUE TITLE
PowerShell Execution Policy - Restricted

### ISSUE THREAD

**Issue by Power-Maverick** [Power-Maverick/PCF-CustomControlBuilder#32]
**Describe the bug**
On some machines where PS execution policy is not set to unrestricted we are getting an error.

**To Reproduce**
1. Set execution policy to default/restricted
2. Click on Build solution
3. ERROR

**Expected behavior**
Error should not occur when building the solution

**Comment by resc-ambit** [comment]
I ran into the same problem even the execution policy was manually changed to unrestricted.
After some time I realized that the field 'MS Build path' in Settings was filled with the output text of the first run while the execution policy was still restricted. This error text contained the message text that msbuild.ps1 cannot be executed due to exectution policy. This error text then was used as MS Build Path in all subsequent calls what caused the user to still see the same text, looking like it was a new error message. 
So the console text was misleading to an improper execution policy.
The **solution** was to clear the content of 'MS Build path' or to manually enter the correct path.
